packages/components/README.md

```diff
@@ -61,10 +61,10 @@ By default, the components are unstyled. This allows you to style the components
 
 ### Including default styles
 
-A set of optional, default styles are included with the components. To include them, import the `styles.css` file from the `@contentful/experience-builder-components` package:
+A set of optional, default styles are included with the components. To include them, import the `styles.css` file from the `@contentful/experiences-components-react` package:
 
 ```jsx
-import '@contentful/experience-builder-components/styles.css';
+import '@contentful/experiences-components-react/styles.css';
```

Comment:
I notice this file doesn't exist in the components package (regardless of the name change)